### ISSUE TITLE
need a good guide or tutorial for using the stig tui interface

### ISSUE THREAD

**Issue by chovy** [rndusr/stig#193]
I think the ? menu is not able to be fully displayed on my small laptop screen is there a way to scroll down?

**Comment by rndusr** [comment]
No, but you can list all keybindings with `stig help keys` or `<?> <k>` in the TUI.

**Comment by chovy** [comment]
Yeah but I’m only seeing the first five.

**Comment by Kabouik** [comment]
You should be able to scroll in the <kbd>?</kbd><kbd>k</kbd> screen with arrow keys: see example with small terminal window height [here](https://asciinema.org/a/1yHpiAzwcviKSqtWSvh4zBayu).
